guides/doc-User_Interface_Quickstart/master.adoc

```diff
@@ -0,0 +1,178 @@
+
+For more information, see {ProvisioningDocURL}creating-compute-profiles_provisioning[Creating Compute Profiles] in _{ProjectName} Provisioning Guide_.
+
+[[fmid-puppet-config-groups]]
+== Puppet Config Groups
+
+// TODO: Modularize?
+//include::modules/con_puppet-introduction.adoc[leveloffset=+1]
+
+* {ManagingHostsDocURL}using-puppet-for-configuration-management_managing-hosts[Using Puppet for Configuration Management]
+* {ManagingHostsDocURL}puppet_guide_using_puppet_classes_managing-hosts[Puppet Classes and Config Groups]
+
+[[fmid-domains]]
+== Domains
+
+// TODO: Modularize this
+Foreman server defines domain names for each host on the network.
+Foreman server must have information about the domain and Smart Proxy server responsible for domain name assignment.
+
+* {ProvisioningDocURL}Configuring_Networking-Adding_a_Domain_to_the_Satellite_Server[Adding a Domain]
+
+[[fmid-partition-tables]]
+== Partition Tables
+
+// TODO: Modularize this
+A partition table is a type of template that defines the way Foreman server configures the disks available on a new host.
+
+* {ProvisioningDocURL}creating-partition-tables_provisioning[Creating Partition Tables]
+
+[[fmid-provisioning-templates]]
+== Provisioning Templates
+
+// TODO: Modularize this
+A provisioning template defines the way Foreman server installs an operating system on a host.
+
+* {ProvisioningDocURL}provisioning-templates_provisioning[Provisioning Templates introduction]
+* {ManagingHostsDocURL}appe-Red_Hat_Satellite-Managing_Hosts-Template_Writing_Reference[Template Writing Reference]
+* {ManagingHostsDocURL}Synchronizing_Templates_Repositories[Synchronizing Template Repositories]
+
+[[fmid-operating-systems]]
+== Operating Systems
+
+// TODO: Modularize this
+An operating system is a collection of resources that define how Foreman server installs a base operating system on a host.
+Operating system entries combine previously defined resources, such as installation media, partition tables, provisioning templates, and others.
+
+* {ProvisioningDocURL}creating-operating-systems_provisioning[Creating Operating Systems]
+
+[[fmid-http-proxies]]
+== HTTP Proxies
+
+If your network uses an HTTP Proxy, you can configure {Project} server to use an HTTP proxy for requests to EC2 Compute Resource.
+ifdef::katello,satellite[]
+Red Hat Content Delivery Network (CDN) or another content sources also use HTTP Proxy definition.
+endif::[]
+
+* {ProvisioningDocURL}Provisioning_Cloud_Instances_in_Amazon_EC2-Using-an-HTTP-Smart-Proxy[Using HTTP Proxy with Amazon EC2]
+* {InstallingProjectDocURL}configuring-satellite-with-an-http-proxy_foreman[Configuring an HTTP Proxy]
+
+// This is completely undocumented, or I haven't found it:
+To configure HTTP Proxy for other Compute Resources, go to *Administer* - *General* and set *HTTP(S) proxy*.
+
+[[fmid-fact-values]]
+== Fact Values
+
+// TODO: This is missing from NG docs: https://theforeman.org/manuals/2.5/index.html#3.5.5FactsandtheENC
```

Comment:
I've opened a new issue; IMHO it should be rather easy to migrate this from the Foreman Manual to the [puppet for configuration management guide](https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#using-puppet-for-configuration-management_managing-hosts). See https://github.com/theforeman/foreman-documentation/issues/579